src/scripts.js

```diff
@@ -438,3 +442,58 @@ var stepTrend = new Chart(ctx, {
     }
   }
 });
+
+// *** EVENT LISTENERS FOR HEADER ***
+$('.toggle label').on('click', function() {
+  if ($(this).siblings().prop('checked')) {
+    changeMode('light');
+  } else {
+    changeMode('dark');
+  }
+  Chart.defaults.global.defaultFontColor = $('body').css('--base-color');
+});
+
+$('.icons li img').on('click', function() {
+  const $widget = $(this).data('type');
+  const $block = $('#user-admin-info');
+  if ($block.data('type') === $widget) {
+    $block.data('type', '').hide();
+  } else {
+    $block.show().children(`.${$widget}-inputs`).show().siblings().hide();
+    $block.data('type', $widget);
+  }
+});
+
+$('#user-admin-info').on('mouseleave', function() {
+  $('#user-admin-info').data('type', '').hide();
+
+});
+
+$('.dropdown header').on('click', function() {
+  $('.dropdown div').toggle();
+});
+
+$('.dropdown div p').on('click', function() {
+  $('.dropdown header p').text($(this).text());
+  $('.dropdown input').val($(this).text());
+  $(this).parent().hide();
+});
```

Comment:
Strong use of this being used which is dope, just want to make sure I understand what this is referring in these contexts. On line 477, guessing this is changing the name of the form to reflect as we click across the icons. Nice work making it dynamic and utilizing this in your favor